### PR TITLE
Widgets mini app

### PR DESCRIPTION
**Pull request by Kevin-Guillen**
https://user-images.githubusercontent.com/84336422/126686063-498f58f6-426d-4015-be38-4b3eebacda3e.mp4

### CODE REVIEW

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by luisjavierglb — APPROVED** [review]
LGTM

File: example/lib/src/models/widget_route.dart [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class WidgetRoute {
+  String title;
+  String widgetRoute;
+
+  WidgetRoute({required this.title, required this.widgetRoute});
```
Comment (FranciscoGaraventa) [inline example/lib/src/models/widget_route.dart]: You missed a trailing comma here

File: example/lib/src/resources/routing.dart [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+              child: Column(
+                children: [
+                  Container(
+                    child: MovieMainInfo(
+                      title: UiConstants.defaultMovieTitle,
+                      releaseDate: UiConstants.defaultReleaseDate,
+                      voteAverage: UiConstants.defaultVoteAverage,
+                    ),
+                  ),
+                ],
```
Comment (MarcosCGlobant) [inline example/lib/src/resources/routing.dart]: why you have a column with just one child?, you could remove it and leave just one container

File: example/lib/src/screen/home_page.dart [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Text(
+                          _widgetList[index].title,
+                          style: TextStyle(
+                            fontSize: UiConstants.containerTitleFontSize,
+                          ),
+                        ),
+                      ],
+                    ),
```
Comment (MarcosCGlobant) [inline example/lib/src/screen/home_page.dart]: who do you have a column with just one child

File: example/lib/src/screen/home_page.dart [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      appBar: AppBar(
+        title: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Text(
+              UiConstants.appBarTitle,
+            ),
+          ],
+        ),
+      ),
```
Comment (MarcosCGlobant) [inline example/lib/src/screen/home_page.dart]: why do you have a column with just one child? you should use the appBar property centerTitle: true, to center the title, and not align it with a column

File: example/lib/src/screen/home_page.dart [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    _widgetList = _generateRoutes();
+  }
+
+  @override
+  Widget build(BuildContext context) {
```
Comment (FranciscoGaraventa) [inline example/lib/src/screen/home_page.dart:57]: why you center three times in this builder?

File: example/lib/src/models/widget_route.dart (hunk 1 above)
Comment (Kevin-Guillen) [reply example/lib/src/models/widget_route.dart]: added

File: example/lib/src/resources/routing.dart (hunk 2 above)
Comment (Kevin-Guillen) [reply example/lib/src/resources/routing.dart]: removed

File: example/lib/src/screen/home_page.dart (hunk 3 above)
Comment (Kevin-Guillen) [reply example/lib/src/screen/home_page.dart]: column removed

File: example/lib/src/screen/home_page.dart (hunk 5 above)
Comment (Kevin-Guillen) [reply example/lib/src/screen/home_page.dart:57]: Removed unused center properties

File: example/lib/src/screen/home_page.dart (hunk 4 above)
Comment (Kevin-Guillen) [reply example/lib/src/screen/home_page.dart]: Column removed.

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**